src/main/java/frc/team2412/robot/commands/climb/ExtendAngledHookCommand.java

```diff
@@ -14,12 +14,12 @@ public ExtendAngledHookCommand(ClimbSubsystem subsystem) {
 
     @Override
     public void initialize() {
-        subsystem.extendAngledArm();
+        subsystem.extendDynamicArm();
```

Comment:
Please either rename the method `extendDynamicArm()` or rename the class `ExtendAngledHookCommand` so that the two match to avoid confusion (same for `RetractAngledHookCommand` and `UnangleClimbHookCommand`).